hips/utils/wcs.py

```diff
@@ -124,6 +126,54 @@ def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str = 'CEL',
 
         return cls(w, shape)
 
+    @classmethod
+    def create_simple(cls, skydir: SkyCoord, shape: tuple, fov: str,
+                      coordsys: str = 'icrs', projection: str = 'AIT') -> 'WCSGeometry':
+        """Create WCS object programmatically using field of view (`WCSGeometry`).
+
+        Parameters
+        ----------
+        skydir : `~astropy.coordinates.SkyCoord`
+            Sky coordinate of the WCS reference point
+        shape : `tuple`
+            Shape of the image (Numpy axis order: y, x)
+        fov: `str`
```

Comment:
@adl1995 - Please address this one remaining inline comment. Otherwise this is 👍  from my side.

@tboch - Assiging to you for final review.